docker/run_worker_queue.sh

```diff
@@ -14,6 +14,10 @@ if test ! -d /var/lib/acme/live/"$QUEUE_ADDRESS"; then
     acmetool want "$QUEUE_ADDRESS"
 fi
 
+# acmetool has a cronjob to reconcile the SSL certificates,
+# so we start the cron daemon
+cron
```

Comment:
You may wish to be [explicit](https://packages.debian.org/buster/amd64/cron/filelist):
```suggestion
/usr/sbin/cron
```